### PR TITLE
Print comments correct in presence of extension points

### PR DESCRIPTION
**Pull request by IwanKaramazow**
Fixes https://github.com/facebook/reason/issues/1938

### CODE REVIEW

**Comment by chenglou** [comment]
Boy you're on fire. Again, I'll let you self review. Ping me when you're done.

**Comment by IwanKaramazow** [comment]
Done 😄It’s low hanging fruit

File: src/reason-parser/reason_pprint_ast.ml [hunk 1]
```diff
@@ -4608,9 +4610,12 @@ let printer = object(self:'self)
         (argsList@[formatJustTheTypeConstraint typeLayout], e)
       | _ -> (argsList, return)
 
-  method bindingsLocationRange l =
+  method bindingsLocationRange ?extension l =
     let len = List.length l in
-    let fstLoc = (List.nth l 0).pvb_loc in
+    let fstLoc = match extension with
+    | Some ext when ext.pexp_loc.loc_ghost == false -> ext.pexp_loc
```
Comment (chenglou) [inline src/reason-parser/reason_pprint_ast.ml]: Can you deep pattern match this one? Probably slightly better than a `when`

**Comment by hcarty** [comment]
Thanks!!!

**Pull request merged**